File: week1/yusuffenes/containsDuplicate.py
def containsDuplicate(self, nums: List[int]) -> bool:
    nums.sort()
    if len(nums)>1:
        for i in range(len(nums)):
            if i<len(nums)-1:
                if nums[i] == nums[i+1]:
                    return True
            else:
                return False
    else:
        return False

File: week1/AhmetTahaSener/ContainsDuplicate.cs
public class Solution
{
    public bool ContainsDuplicate(int[] nums)
    {
        Array.Sort(nums);
        for (int i = 1; i < nums.Length; i++)
        {
            if (nums[i - 1] == nums[i])
            {
                return true;
            }
        }
        return false;
    }
}